lib/themeSchema.schema.json

```diff
@@ -0,0 +1,199 @@
+{
```

Comment:
Would prefer to put this in `config/schemas/themeConfig.json` and move `lib/themeConfig.schema.json` as well 🍹